daisy_workflows/image_import/windows/translate.ps1

```diff
@@ -44,14 +44,17 @@ function Get-MetadataValue {
   try {
     $client = New-Object Net.WebClient
```

Comment:
Thoughts on retrying failures?

Reply:
That's a great idea. I added 3 retries and a 10 second sleep between each attempt.